README.md

```diff
@@ -1,2 +1,26 @@
-# Intelligent_placer
-University assignment: python lib that check if items in photo can be placed into figure nearby items
+# Intelligent placer
+Given a picture of several objects on a light horizontal surface and a polygon as input, determine whether it is possible to place all these objects simultaneously on the plane so that they fit into the polygon.
+
+## Task definition
+Algorithm input: 
+- The path to a file with tif, png or jpg extension containing a three-channel RGB image on which on the horizontal light surface there is a sheet of A4 paper with a polygon drawn on the left and objects from a known set on the right;
+- The path to the file with the same extension as in the previous paragraph, which contains an image of the surface on which a sheet of paper with a polygon and items will be placed;
+- 4 integers describing the area within which the integer objects we are going to consider are contained. the first and second numbers represent the start and end pixel (inclusive) horizontally, and the second and third numbers represent the start and end pixel of the area vertically.
+Algorithm output:
+- True if goods can be placed in a polygon, otherwise false.
+
+## Input image requirements
+- Objects cannot intersect;
+- There must be a distance of at least 10 pixels between objects;
+- The border of a polygon must be at least 4 pixels thick at its narrowest point;
+- The smaller of the two visible object dimensions must be at least 20 pixels;
+- The larger of the two visible object dimensions must be at least 100 pixels;
+- The angle of inclination of the chamber from the surface normal should be from 80 to 100 degrees;
```

Comment:
I think this sentence means the angle of deviation from the normal within 10 degrees.
Because otherwise it says here that we can shoot under the surface.
![image](https://user-images.githubusercontent.com/60853085/154768809-d5489db2-97ba-4cf9-ae0b-d7539c460766.png)

Reply:
Thanks! Yes, you are right 😄 Fixed